dev/domain-controller/new-admin-vm/Install-CAM.ps1

```diff
@@ -1383,6 +1383,7 @@ brokerLocale=en_US
 "@
 					[System.Net.ServicePointManager]::CertificatePolicy = New-Object TrustAllCertsPolicy
 				}
+                [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
```

Comment:
Do we have the indenting right? I hate indenting...

Reply:
i should have looked at it here... we do not. I will fix it. One is using tabs and the other spaces